### ISSUE TITLE
the raw phenotype indeed influence the kmers passed the threshold。

### ISSUE THREAD

**Issue by laserhsu** [voichek/kmersGWAS#113]
Dear sir，
Thanks for your great work.
I have run the pipeline many times and have found an interesting thing. The pre-processing of the phenotypic data influnence the significant kmers. When I used the raw phenotype, I have found 10k  significant kmers as I have asked in issues #107 . But when I did the standardization (calculating the zscores of phenotypes), than the results turned to be just several significant kmers. So I wonder, since your have done 2k traits in your papers, do you do standardization? is it necessary to do some pre-processing? After all, the results are different. 

There are the head lines of my phenotypes and their zscores. 
![图片](https://user-images.githubusercontent.com/43374999/160081540-57d8fcf2-b2c3-4c77-8213-7d297f813766.png)

![图片](https://user-images.githubusercontent.com/43374999/160081494-b7ee95e5-937c-4db3-b76c-87089315b982.png)

Best,
Han Xu

**Comment by voichek** [comment]
Dear Han Xu,

First, the two snapshots you posted don't make sense to me, for example the three first values in the first one are high (34-39) and similar to each other and after transformation they are very different (2,-0.5,0.2). Am I missing something?

It seems from your first snapshot that you have quite a bimodal distribution, so it makes sense that results would be affected by the transformation. This is a general question on GWAS (not specific for k-mers), so maybe there are better sources than me to consult. 

As I mentioned before, I think QQ plot is a good indication if transformation of phenotype or other modifications are needed.

In the 2000 phenotypes in the paper I decided not to transform and use data as reported in the original paper, to try and be consistent. In another project I tried to run the k-mer-GWAS on 10K+ gene-expression phenotypes (eQTL study) and there I did transform values, but I think it was less extreme than your situation here as you seem to have a bimodal distribution.

Hope it helps,
Yoav
